2018-09-29 14:10:05.730 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:10:05.732 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:10:05.777 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:10:05.778 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:10:05.779 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:10:06.812 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.033s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:10:06.815 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:10:07.772 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.958s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:10:07.774 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:10:08.565 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.791s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:10:08.568 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:10:09.339 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.772s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:11:05.732 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:11:05.734 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:11:05.768 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:11:05.769 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:11:05.771 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:11:06.748 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 0.978s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:11:06.751 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:11:07.527 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.777s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:11:07.529 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:11:08.253 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.723s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:11:08.255 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:11:08.959 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.705s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:12:05.732 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:12:05.735 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:12:05.768 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:12:05.769 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:12:05.770 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:12:06.552 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 0.782s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:12:06.555 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:12:07.300 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.745s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:12:07.303 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:12:08.020 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.718s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:12:08.022 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:12:08.783 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.761s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:05.735 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:13:05.739 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:13:05.823 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:13:05.825 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:13:05.827 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:07.096 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.269s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:07.100 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:08.064 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.964s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:08.068 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:09.011 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.944s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:09.014 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:09.909 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.896s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:22.701 6036 DEBUG cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Volume reschedule parameters: True retry: {u'num_attempts': 1, u'backends': [u'localhost.localdomain@lvm#lvm'], u'hosts': [u'localhost.localdomain@lvm#lvm']} get_flow /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:1126
2018-09-29 14:13:22.735 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Flow 'volume_create_manager' (cc61d039-1419-44ea-b032-9b9ef5d00357) transitioned into state 'RUNNING' from state 'PENDING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-09-29 14:13:22.741 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (ac80324e-1c36-4e10-a975-c55210be8919) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-29 14:13:22.786 6036 DEBUG oslo_db.sqlalchemy.engines [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-09-29 14:13:22.823 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeRefTask;volume:create' (ac80324e-1c36-4e10-a975-c55210be8919) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'Volume(_name_id=None,admin_metadata={},attach_status='detached',availability_zone='nova',bootable=False,cluster=<?>,cluster_name=None,consistencygroup=<?>,consistencygroup_id=None,created_at=2018-09-29T18:13:22Z,deleted=False,deleted_at=None,display_description=None,display_name='tempest-VOLUME_SAMPLE-1538244567',ec2_id=None,encryption_key_id=None,glance_metadata=<?>,group=<?>,group_id=None,host='localhost.localdomain@lvm#lvm',id=846f1130-37b7-46af-bae2-270972ced610,launched_at=None,metadata={},migration_status=None,multiattach=False,previous_status=None,project_id='36962c6ef4284c22a92dddcad655b144',provider_auth=None,provider_geometry=None,provider_id=None,provider_location=None,replication_driver_data=None,replication_extended_status=None,replication_status=None,scheduled_at=2018-09-29T18:13:22Z,size=1,snapshot_id=None,snapshots=<?>,source_volid=None,status='creating',terminated_at=None,updated_at=2018-09-29T18:13:22Z,user_id='b8a307e91dbc4360a59951873e17e25e',volume_attachment=<?>,volume_type=VolumeType(f7f6fe9b-db4d-462d-ba3e-1b81bec5277f),volume_type_id=f7f6fe9b-db4d-462d-ba3e-1b81bec5277f)' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-29 14:13:22.828 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (f9be03bb-b924-4863-94cd-5fe270964bf1) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-29 14:13:22.831 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.OnFailureRescheduleTask;volume:create' (f9be03bb-b924-4863-94cd-5fe270964bf1) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-29 14:13:22.833 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (d61f3577-7e9d-4d16-a40e-00a2f797dc5a) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-29 14:13:22.835 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:22.867 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:22.868 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:22 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1538244567", "tags": [], "container_format": "bare", "created_at": "2018-09-29T18:10:09Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-29T18:10:09Z", "visibility": "public", "self": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b", "min_disk": 0, "protected": false, "id": "d01c7db9-7442-441e-ad24-681b73c8754b", "file": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "63d6ee59c98741e4acf874cc7755dc45", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:22.870 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:22.880 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/schemas/image used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:22.882 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:22 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:22.886 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:22.923 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:22.925 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:22 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1538244567", "tags": [], "container_format": "bare", "created_at": "2018-09-29T18:10:09Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-29T18:10:09Z", "visibility": "public", "self": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b", "min_disk": 0, "protected": false, "id": "d01c7db9-7442-441e-ad24-681b73c8754b", "file": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "63d6ee59c98741e4acf874cc7755dc45", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:22.926 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:22.941 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/schemas/image used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:22.942 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:22 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:22.945 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:22.957 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/schemas/image used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:22.958 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:22 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:22.961 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.ExtractVolumeSpecTask;volume:create' (d61f3577-7e9d-4d16-a40e-00a2f797dc5a) transitioned into state 'SUCCESS' from state 'RUNNING' with result '{'status': u'creating', 'image_location': (None, None), 'image_id': u'd01c7db9-7442-441e-ad24-681b73c8754b', 'volume_id': u'846f1130-37b7-46af-bae2-270972ced610', 'volume_size': 1, 'image_service': <cinder.image.glance.GlanceImageService object at 0x7fbde0ba3d10>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1538244567', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 9, 29, 18, 10, 9, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 9, 29, 18, 10, 9, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'63d6ee59c98741e4acf874cc7755dc45', u'protected': False, u'id': u'd01c7db9-7442-441e-ad24-681b73c8754b', u'file': u'/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}, 'volume_name': u'volume-846f1130-37b7-46af-bae2-270972ced610', 'type': 'image'}' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-29 14:13:22.965 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (720bdb6b-5d45-4658-a915-2e37b4969f1e) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-29 14:13:22.998 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.NotifyVolumeActionTask;volume:create, create.start' (720bdb6b-5d45-4658-a915-2e37b4969f1e) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-29 14:13:23.001 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (5556a8f1-2fbf-4ff7-adda-f167d5539cd0) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-29 14:13:23.002 6036 INFO cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Volume 846f1130-37b7-46af-bae2-270972ced610: being created as image with specification: {'status': u'creating', 'image_location': (None, None), 'volume_size': 1, 'volume_name': u'volume-846f1130-37b7-46af-bae2-270972ced610', 'image_id': u'd01c7db9-7442-441e-ad24-681b73c8754b', 'image_service': <cinder.image.glance.GlanceImageService object at 0x7fbde0ba3d10>, 'image_meta': {u'status': u'active', u'name': u'tempest-cirros-0.4.0-x86_64-1538244567', u'tags': [], u'container_format': u'bare', u'created_at': datetime.datetime(2018, 9, 29, 18, 10, 9, tzinfo=<iso8601.Utc>), u'disk_format': u'qcow2', u'updated_at': datetime.datetime(2018, 9, 29, 18, 10, 9, tzinfo=<iso8601.Utc>), u'visibility': u'public', 'properties': {}, u'owner': u'63d6ee59c98741e4acf874cc7755dc45', u'protected': False, u'id': u'd01c7db9-7442-441e-ad24-681b73c8754b', u'file': u'/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file', u'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', u'min_disk': 0, u'virtual_size': None, u'min_ram': 0, u'size': 12716032}}
2018-09-29 14:13:23.003 6036 DEBUG cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Cloning 846f1130-37b7-46af-bae2-270972ced610 from image d01c7db9-7442-441e-ad24-681b73c8754b  at location (None, None). _create_from_image /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:896
2018-09-29 14:13:23.005 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:23.041 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:23.042 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Type: application/octet-stream
Content-Md5: 443b7623e27ecf03dc9e01ee93f67afe
Content-Length: 12716032
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:23 GMT
Connection: keep-alive
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:23.153 6036 DEBUG cinder.image.image_utils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Image fetch details: dest /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm, size 12.00 MB, duration 1.00 sec fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:247
2018-09-29 14:13:23.154 6036 INFO cinder.image.image_utils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Image download 12.00 MB at 12.00 MB/s
2018-09-29 14:13:23.155 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:23.206 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:23.208 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:23 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1538244567", "tags": [], "container_format": "bare", "created_at": "2018-09-29T18:10:09Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-29T18:10:09Z", "visibility": "public", "self": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b", "min_disk": 0, "protected": false, "id": "d01c7db9-7442-441e-ad24-681b73c8754b", "file": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "63d6ee59c98741e4acf874cc7755dc45", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:23.210 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:23.223 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/schemas/image used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:23.224 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:23 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:23.227 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:24.042 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm" returned: 0 in 0.814s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:24.043 6036 DEBUG cinder.image.image_utils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Temporary image d01c7db9-7442-441e-ad24-681b73c8754b is fetched for user b8a307e91dbc4360a59951873e17e25e. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:672
2018-09-29 14:13:24.045 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:24.986 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm" returned: 0 in 0.942s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:24.989 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-846f1130-37b7-46af-bae2-270972ced610 cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:26.057 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvcreate -T -V 1g -n volume-846f1130-37b7-46af-bae2-270972ced610 cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.068s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:26.081 6036 DEBUG cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Attempting download of d01c7db9-7442-441e-ad24-681b73c8754b ((None, None)) to volume 846f1130-37b7-46af-bae2-270972ced610. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:536
2018-09-29 14:13:26.084 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:26.518 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:26.520 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Length: 616
Content-Type: application/json
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:26 GMT
Connection: keep-alive

{"status": "active", "name": "tempest-cirros-0.4.0-x86_64-1538244567", "tags": [], "container_format": "bare", "created_at": "2018-09-29T18:10:09Z", "size": 12716032, "disk_format": "qcow2", "updated_at": "2018-09-29T18:10:09Z", "visibility": "public", "self": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b", "min_disk": 0, "protected": false, "id": "d01c7db9-7442-441e-ad24-681b73c8754b", "file": "/v2/images/d01c7db9-7442-441e-ad24-681b73c8754b/file", "checksum": "443b7623e27ecf03dc9e01ee93f67afe", "owner": "63d6ee59c98741e4acf874cc7755dc45", "virtual_size": null, "min_ram": 0, "schema": "/v2/schemas/image"}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:26.522 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] curl -g -i -X GET -H 'Accept-Encoding: gzip, deflate' -H 'Accept: */*' -H 'X-Auth-Token: {SHA1}41f4273e1427d622e93b32b97f051e3f87e1ee79' -H 'Connection: keep-alive' -H 'User-Agent: python-glanceclient' -H 'Content-Type: application/octet-stream' -H 'X-OpenStack-Request-ID: req-7958354b-85e2-4817-9e13-f52cd3e831cf' http://10.0.20.36:9292/v2/schemas/image log_curl_request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:188
2018-09-29 14:13:26.536 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] GET call to image for http://10.0.20.36:9292/v2/schemas/image used request id req-7958354b-85e2-4817-9e13-f52cd3e831cf _request /usr/lib/python2.7/site-packages/glanceclient/common/http.py:276
2018-09-29 14:13:26.538 6036 DEBUG glanceclient.common.http [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] 
HTTP/1.1 200 OK
Content-Type: application/json
Content-Length: 4157
X-Openstack-Request-Id: req-7958354b-85e2-4817-9e13-f52cd3e831cf
Date: Sat, 29 Sep 2018 18:13:26 GMT
Connection: keep-alive

{"additionalProperties": {"type": "string"}, "name": "image", "links": [{"href": "{self}", "rel": "self"}, {"href": "{file}", "rel": "enclosure"}, {"href": "{schema}", "rel": "describedby"}], "properties": {"status": {"readOnly": true, "enum": ["queued", "saving", "active", "killed", "deleted", "pending_delete", "deactivated"], "type": "string", "description": "Status of the image"}, "tags": {"items": {"type": "string", "maxLength": 255}, "type": "array", "description": "List of strings related to the image"}, "kernel_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the kernel when booting an AMI-style image.", "is_base": false}, "container_format": {"enum": [null, "ami", "ari", "aki", "bare", "ovf", "ova", "docker"], "type": ["null", "string"], "description": "Format of the container"}, "min_ram": {"type": "integer", "description": "Amount of ram (in MB) required to boot image."}, "ramdisk_id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": ["null", "string"], "description": "ID of image stored in Glance that should be used as the ramdisk when booting an AMI-style image.", "is_base": false}, "locations": {"items": {"required": ["url", "metadata"], "type": "object", "properties": {"url": {"type": "string", "maxLength": 255}, "metadata": {"type": "object"}}}, "type": "array", "description": "A set of URLs to access the image file kept in external store"}, "visibility": {"enum": ["community", "public", "private", "shared"], "type": "string", "description": "Scope of image accessibility"}, "updated_at": {"readOnly": true, "type": "string", "description": "Date and time of the last image modification"}, "owner": {"type": ["null", "string"], "description": "Owner of the image", "maxLength": 255}, "file": {"readOnly": true, "type": "string", "description": "An image file url"}, "min_disk": {"type": "integer", "description": "Amount of disk space (in GB) required to boot image."}, "virtual_size": {"readOnly": true, "type": ["null", "integer"], "description": "Virtual size of image in bytes"}, "id": {"pattern": "^([0-9a-fA-F]){8}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){4}-([0-9a-fA-F]){12}$", "type": "string", "description": "An identifier for the image"}, "size": {"readOnly": true, "type": ["null", "integer"], "description": "Size of image file in bytes"}, "instance_uuid": {"type": "string", "description": "Metadata which can be used to record which instance this image is associated with. (Informational only, does not create an instance snapshot.)", "is_base": false}, "os_distro": {"type": "string", "description": "Common name of operating system distribution as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "name": {"type": ["null", "string"], "description": "Descriptive name for the image", "maxLength": 255}, "checksum": {"readOnly": true, "type": ["null", "string"], "description": "md5 hash of image contents.", "maxLength": 32}, "created_at": {"readOnly": true, "type": "string", "description": "Date and time of image registration"}, "disk_format": {"enum": [null, "ami", "ari", "aki", "vhd", "vhdx", "vmdk", "raw", "qcow2", "vdi", "iso", "ploop"], "type": ["null", "string"], "description": "Format of the disk"}, "os_version": {"type": "string", "description": "Operating system version as specified by the distributor", "is_base": false}, "protected": {"type": "boolean", "description": "If true, image will not be deletable."}, "architecture": {"type": "string", "description": "Operating system architecture as specified in https://docs.openstack.org/python-glanceclient/latest/cli/property-keys.html", "is_base": false}, "direct_url": {"readOnly": true, "type": "string", "description": "URL to access the image file kept in external store"}, "self": {"readOnly": true, "type": "string", "description": "An image self url"}, "schema": {"readOnly": true, "type": "string", "description": "An image schema url"}}}
 log_http_response /usr/lib/python2.7/site-packages/glanceclient/common/http.py:202
2018-09-29 14:13:26.543 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmphHtD1_ execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:27.466 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmphHtD1_" returned: 0 in 0.923s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:27.469 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:28.615 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm" returned: 0 in 1.146s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:28.617 6036 DEBUG cinder.image.image_utils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] d01c7db9-7442-441e-ad24-681b73c8754b was qcow2, converting to raw  fetch_to_volume_format /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:417
2018-09-29 14:13:28.619 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--846f1130--37b7--46af--bae2--270972ced610 oflag=direct execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:29.554 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf dd count=0 if=/var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm of=/dev/mapper/cinder--volumes2-volume--846f1130--37b7--46af--bae2--270972ced610 oflag=direct" returned: 0 in 0.936s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:29.556 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--846f1130--37b7--46af--bae2--270972ced610 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:30.855 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf qemu-img convert -t none -f qcow2 -O raw /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm /dev/mapper/cinder--volumes2-volume--846f1130--37b7--46af--bae2--270972ced610" returned: 0 in 1.299s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:30.857 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): /usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:31.914 6036 DEBUG oslo_concurrency.processutils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "/usr/bin/python2 -m oslo_concurrency.prlimit --as=1073741824 --cpu=8 -- sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C qemu-img info /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm" returned: 0 in 1.056s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:31.916 6036 DEBUG cinder.image.image_utils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Image conversion details: src /var/lib/cinder/conversion/tmpuMr8Urlocalhost.localdomain@lvm, size 44.00 MB, duration 1.30 sec, destination /dev/mapper/cinder--volumes2-volume--846f1130--37b7--46af--bae2--270972ced610 _convert_image /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:190
2018-09-29 14:13:31.917 6036 INFO cinder.image.image_utils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Converted 44.00 MB image at 33.81 MB/s
2018-09-29 14:13:31.919 6036 DEBUG cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Downloaded image d01c7db9-7442-441e-ad24-681b73c8754b ((None, None)) to volume 846f1130-37b7-46af-bae2-270972ced610 successfully. _copy_image_to_volume /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:582
2018-09-29 14:13:31.923 6036 DEBUG cinder.image.image_utils [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Temporary image d01c7db9-7442-441e-ad24-681b73c8754b for user b8a307e91dbc4360a59951873e17e25e is deleted. fetch /usr/lib/python2.7/site-packages/cinder/image/image_utils.py:676
2018-09-29 14:13:31.925 6036 DEBUG cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Marking volume 846f1130-37b7-46af-bae2-270972ced610 as bootable. _enable_bootable_flag /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:479
2018-09-29 14:13:31.936 6036 DEBUG cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Copying metadata from image d01c7db9-7442-441e-ad24-681b73c8754b to 846f1130-37b7-46af-bae2-270972ced610. _handle_bootable_volume_glance_meta /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:437
2018-09-29 14:13:31.937 6036 DEBUG cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Creating volume glance metadata for volume 846f1130-37b7-46af-bae2-270972ced610 backed by image d01c7db9-7442-441e-ad24-681b73c8754b with: {'container_format': u'bare', 'min_ram': 0, 'disk_format': u'qcow2', 'image_name': u'tempest-cirros-0.4.0-x86_64-1538244567', 'image_id': u'd01c7db9-7442-441e-ad24-681b73c8754b', 'checksum': u'443b7623e27ecf03dc9e01ee93f67afe', 'min_disk': 0, 'size': 12716032}. _capture_volume_image_metadata /usr/lib/python2.7/site-packages/cinder/volume/flows/manager/create_volume.py:616
2018-09-29 14:13:32.022 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeFromSpecTask;volume:create' (5556a8f1-2fbf-4ff7-adda-f167d5539cd0) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-29 14:13:32.025 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (cdad4578-cd14-43de-a5da-2d5c85152105) transitioned into state 'RUNNING' from state 'PENDING' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:194
2018-09-29 14:13:32.074 6036 INFO cinder.volume.flows.manager.create_volume [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Volume volume-846f1130-37b7-46af-bae2-270972ced610 (846f1130-37b7-46af-bae2-270972ced610): created successfully
2018-09-29 14:13:32.077 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Task 'cinder.volume.flows.manager.create_volume.CreateVolumeOnFinishTask;volume:create, create.end' (cdad4578-cd14-43de-a5da-2d5c85152105) transitioned into state 'SUCCESS' from state 'RUNNING' with result 'None' _task_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:183
2018-09-29 14:13:32.081 6036 DEBUG cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Flow 'volume_create_manager' (cc61d039-1419-44ea-b032-9b9ef5d00357) transitioned into state 'SUCCESS' from state 'RUNNING' _flow_receiver /usr/lib/python2.7/site-packages/taskflow/listeners/logging.py:145
2018-09-29 14:13:32.082 6036 INFO cinder.volume.manager [req-7958354b-85e2-4817-9e13-f52cd3e831cf b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Created volume successfully.
2018-09-29 14:13:54.850 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:55.785 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvchange -a y --yes -K cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610" returned: 0 in 0.935s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:55.842 6036 INFO cinder.volume.targets.lio [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Creating iscsi_target for volume: volume-846f1130-37b7-46af-bae2-270972ced610
2018-09-29 14:13:55.843 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:13:55.844 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610 iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 tk4LHgHxh35jeQ9G9L7f b5C4qpSaGhA34PP7 False -p3260 -a10.0.20.36 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:57.310 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool create /dev/cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610 iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 tk4LHgHxh35jeQ9G9L7f b5C4qpSaGhA34PP7 False -p3260 -a10.0.20.36" returned: 0 in 1.466s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:57.313 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.470s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:13:57.314 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:13:57.316 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:13:58.924 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 1.609s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:13:58.927 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.612s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:13:58.928 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:13:58.929 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:14:00.330 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 1.402s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:14:00.333 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.405s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:14:00.334 6036 DEBUG cinder.volume.targets.iscsi [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Set provider_location to: 10.0.20.36:3260,iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 0 create_export /usr/lib/python2.7/site-packages/cinder/volume/targets/iscsi.py:215
2018-09-29 14:14:00.345 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:14:00.346 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 tk4LHgHxh35jeQ9G9L7f b5C4qpSaGhA34PP7 iqn.1994-05.com.redhat:ac38aeb3e7f0 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:14:02.147 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool add-initiator iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 tk4LHgHxh35jeQ9G9L7f b5C4qpSaGhA34PP7 iqn.1994-05.com.redhat:ac38aeb3e7f0" returned: 0 in 1.801s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:14:02.150 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.805s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:14:02.153 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:14:02.155 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:14:04.425 6036 DEBUG oslo_concurrency.processutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 2.270s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:14:04.428 6036 DEBUG oslo_concurrency.lockutils [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.275s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:14:04.473 6036 INFO cinder.volume.manager [req-259ec30e-351f-44b7-b22c-ff872caa7538 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Initialize volume connection completed successfully.
2018-09-29 14:14:05.736 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:14:05.738 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:14:05.772 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:14:05.773 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:14:05.775 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:14:06.904 6036 DEBUG cinder.coordination [req-2132306b-3546-4896-82b0-41c75f250c75 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "/var/lib/cinder/cinder-846f1130-37b7-46af-bae2-270972ced610" acquired by "attach_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-29 14:14:07.100 6036 INFO cinder.volume.manager [req-2132306b-3546-4896-82b0-41c75f250c75 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Attaching volume 846f1130-37b7-46af-bae2-270972ced610 to instance 5e434d75-e869-4e1a-ae6c-58e58b369eef at mountpoint /dev/vdb on host None.
2018-09-29 14:14:07.274 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.500s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:14:07.276 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:14:07.376 6036 INFO cinder.volume.manager [req-2132306b-3546-4896-82b0-41c75f250c75 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Attach volume completed successfully.
2018-09-29 14:14:07.378 6036 DEBUG cinder.coordination [req-2132306b-3546-4896-82b0-41c75f250c75 b8a307e91dbc4360a59951873e17e25e 36962c6ef4284c22a92dddcad655b144 - 5a478b50973a4ec48ed9f4dcd764ca88 5a478b50973a4ec48ed9f4dcd764ca88] Lock "/var/lib/cinder/cinder-846f1130-37b7-46af-bae2-270972ced610" released by "attach_volume" :: held 0.474s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-29 14:14:08.206 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.930s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:14:08.209 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:14:09.238 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.030s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:14:09.241 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:14:10.221 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.980s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:05.735 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:15:05.737 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:15:05.769 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:15:05.770 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:15:05.771 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:06.902 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.131s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:06.906 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:07.667 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.761s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:07.670 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:08.467 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.797s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:08.470 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:09.197 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.728s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:43.520 6036 DEBUG oslo_concurrency.lockutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:15:43.522 6036 DEBUG oslo_concurrency.processutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 iqn.1994-05.com.redhat:ac38aeb3e7f0 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:45.216 6036 DEBUG oslo_concurrency.processutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete-initiator iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 iqn.1994-05.com.redhat:ac38aeb3e7f0" returned: 0 in 1.693s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:45.217 6036 DEBUG oslo_concurrency.lockutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.697s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:15:45.218 6036 DEBUG oslo_concurrency.lockutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:15:45.219 6036 DEBUG oslo_concurrency.processutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:47.123 6036 DEBUG oslo_concurrency.processutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 1.904s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:47.125 6036 DEBUG oslo_concurrency.lockutils [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.907s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:15:47.126 6036 INFO cinder.volume.manager [req-308ac31e-3854-441c-9320-4d684dbf78cb e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Terminate volume connection completed successfully.
2018-09-29 14:15:47.437 6036 DEBUG cinder.coordination [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "/var/lib/cinder/cinder-846f1130-37b7-46af-bae2-270972ced610-detach_volume" acquired by "detach_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-29 14:15:47.517 6036 INFO cinder.volume.manager [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Detaching volume 846f1130-37b7-46af-bae2-270972ced610 from instance 5e434d75-e869-4e1a-ae6c-58e58b369eef.
2018-09-29 14:15:47.518 6036 DEBUG oslo_concurrency.lockutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:15:47.520 6036 DEBUG oslo_concurrency.processutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:49.465 6036 DEBUG oslo_concurrency.processutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 1.945s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:49.468 6036 DEBUG oslo_concurrency.lockutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.949s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:15:49.469 6036 INFO cinder.volume.targets.lio [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Removing iscsi_target: 846f1130-37b7-46af-bae2-270972ced610
2018-09-29 14:15:49.471 6036 DEBUG oslo_concurrency.lockutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:15:49.473 6036 DEBUG oslo_concurrency.processutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:51.200 6036 DEBUG oslo_concurrency.processutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool delete iqn.2010-10.org.openstack:volume-846f1130-37b7-46af-bae2-270972ced610" returned: 0 in 1.728s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:51.202 6036 DEBUG oslo_concurrency.lockutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.731s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:15:51.203 6036 DEBUG oslo_concurrency.lockutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:15:51.205 6036 DEBUG oslo_concurrency.processutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:15:53.013 6036 DEBUG oslo_concurrency.processutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool save" returned: 0 in 1.808s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:15:53.015 6036 DEBUG oslo_concurrency.lockutils [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 1.812s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:15:53.188 6036 INFO cinder.volume.manager [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Detach volume completed successfully.
2018-09-29 14:15:53.189 6036 DEBUG cinder.coordination [req-d15fea14-ab9a-49b2-977a-92de65febd36 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "/var/lib/cinder/cinder-846f1130-37b7-46af-bae2-270972ced610-detach_volume" released by "detach_volume" :: held 5.751s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-29 14:16:00.089 6036 DEBUG cinder.coordination [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "/var/lib/cinder/cinder-846f1130-37b7-46af-bae2-270972ced610-delete_volume" acquired by "delete_volume" :: waited 0.001s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:175
2018-09-29 14:16:00.246 6036 DEBUG oslo_concurrency.lockutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" acquired by "cinder.volume.targets.lio._execute" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-09-29 14:16:00.249 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:02.731 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf cinder-rtstool get-targets" returned: 0 in 2.482s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:02.733 6036 DEBUG oslo_concurrency.lockutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "lioadm" released by "cinder.volume.targets.lio._execute" :: held 2.487s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-09-29 14:16:02.734 6036 INFO cinder.volume.targets.iscsi [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Skipping remove_export. No iscsi_target is presently exported for volume: 846f1130-37b7-46af-bae2-270972ced610
2018-09-29 14:16:02.737 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:03.884 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610" returned: 0 in 1.148s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:03.887 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:05.002 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvdisplay --noheading -C -o Attr cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610" returned: 0 in 1.116s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:05.006 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:05.731 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:16:05.734 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:16:05.781 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:16:05.781 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:16:05.783 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:06.422 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf lvremove --config activation { retry_deactivation = 1}  -f cinder-volumes2/volume-846f1130-37b7-46af-bae2-270972ced610" returned: 0 in 1.417s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:06.426 6036 INFO cinder.volume.drivers.lvm [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Successfully deleted volume: 846f1130-37b7-46af-bae2-270972ced610
2018-09-29 14:16:06.510 6036 WARNING cinder.quota [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Deprecated: Default quota for resource: gigabytes_iscsi is set by the default quota flag: quota_gigabytes_iscsi, it is now deprecated. Please use the default quota class for default quota.
2018-09-29 14:16:06.512 6036 WARNING cinder.quota [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Deprecated: Default quota for resource: volumes_iscsi is set by the default quota flag: quota_volumes_iscsi, it is now deprecated. Please use the default quota class for default quota.
2018-09-29 14:16:06.558 6036 DEBUG cinder.quota [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Created reservations ['27ea9277-41aa-48a1-9389-28c061f16d25', '60651344-2e20-400b-adaa-5ca38392132a', '13bb0f86-b59f-49f9-bd49-db97af117218', '343a27a2-12c4-431e-859f-3dbb7ae984aa'] reserve /usr/lib/python2.7/site-packages/cinder/quota.py:1029
2018-09-29 14:16:06.643 6036 DEBUG cinder.volume.utils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Can not find volume 846f1130-37b7-46af-bae2-270972ced610 at notify usage _usage_from_volume /usr/lib/python2.7/site-packages/cinder/volume/utils.py:99
2018-09-29 14:16:06.717 6036 DEBUG cinder.volume.drivers.lvm [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:16:06.720 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:06.864 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.081s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:06.867 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:07.958 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.239s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:07.962 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:08.051 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.185s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:08.054 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:08.872 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.910s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:08.875 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:09.080 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 1.027s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:09.084 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:09.803 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.928s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:09.806 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:16:10.144 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 1.060s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:10.701 6036 DEBUG oslo_concurrency.processutils [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.896s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:16:10.703 6036 DEBUG cinder.manager [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:16:11.042 6036 INFO cinder.volume.manager [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Deleted volume successfully.
2018-09-29 14:16:11.052 6036 DEBUG cinder.coordination [req-8b49877b-fddc-4f71-9389-f2295b2b2801 e386cb43209f41fcb7173020f656e97e 63d6ee59c98741e4acf874cc7755dc45 - default default] Lock "/var/lib/cinder/cinder-846f1130-37b7-46af-bae2-270972ced610-delete_volume" released by "delete_volume" :: held 10.963s _synchronized /usr/lib/python2.7/site-packages/cinder/coordination.py:187
2018-09-29 14:17:05.728 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._publish_service_capabilities run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:17:05.731 6036 DEBUG cinder.manager [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Notifying Schedulers of capabilities ... _publish_service_capabilities /usr/lib/python2.7/site-packages/cinder/manager.py:189
2018-09-29 14:17:05.754 6036 DEBUG oslo_service.periodic_task [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running periodic task VolumeManager._report_driver_status run_periodic_tasks /usr/lib/python2.7/site-packages/oslo_service/periodic_task.py:215
2018-09-29 14:17:05.755 6036 DEBUG cinder.volume.drivers.lvm [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Updating volume stats _update_volume_stats /usr/lib/python2.7/site-packages/cinder/volume/drivers/lvm.py:205
2018-09-29 14:17:05.756 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:17:06.804 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C vgs --noheadings --unit=g -o name,size,free,lv_count,uuid --separator : --nosuffix cinder-volumes2" returned: 0 in 1.048s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:17:06.807 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:17:07.665 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.858s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:17:07.667 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:17:08.373 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o size,data_percent --separator : --nosuffix /dev/cinder-volumes2/cinder-volumes2-pool" returned: 0 in 0.706s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
2018-09-29 14:17:08.375 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] Running cmd (subprocess): sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2 execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:367
2018-09-29 14:17:09.118 6036 DEBUG oslo_concurrency.processutils [req-5c3397ea-debb-40fd-9642-b2681d6fed51 - - - - -] CMD "sudo cinder-rootwrap /etc/cinder/rootwrap.conf env LC_ALL=C lvs --noheadings --unit=g -o vg_name,name,size --nosuffix cinder-volumes2" returned: 0 in 0.743s execute /usr/lib/python2.7/site-packages/oslo_concurrency/processutils.py:404
